src/local-fs.ts

```diff
@@ -195,7 +195,8 @@ export default class LocalFS implements ILocalFSPackageManager {
 
     const pathName: string = this._getStorage(name);
 
-    fs.exists(pathName, exists => {
+    fs.access(pathName, (fileNotFound) => {
+      const exists = !fileNotFound;
```

Comment:
The approach displayed in the docs seems much more easy to understand, what do you think?
https://nodejs.org/api/fs.html#fs_fs_access_path_mode_callback
```
// Check if the file exists in the current directory.
fs.access(file, fs.constants.F_OK, (err) => {
  console.log(`${file} ${err ? 'does not exist' : 'exists'}`);
});
```

Reply:
This implementation is very similar to the linked one. The fs.constants.F_OK is redundant in the original example, since it is the default mode, so I left it out. I gave the error a name because I thought it made the logic easier to follow. If you know that the error case is file-not-found, it is clear that the opposite of that is that the file exists. Since I didn't want to flip the if-else statement that follows, I had to negate the condition, otherwise we'd have to check against `!fileNotFound` which in my opinion is less intuitive.

Reply:
Thanks @juanmanual 👍

Reply:
One more question, about this

https://nodejs.org/api/fs.html#fs_fs_access_path_mode_callback

> Using fs.access() to check for the accessibility of a file before calling fs.open(), fs.readFile() or fs.writeFile() is not recommended. Doing so introduces a race condition, since other processes may change the file's state between the two calls. Instead, user code should open/read/write the file directly and handle the error raised if the file is not accessible.

We follow the `fs.access` with a `const file = fs.createWriteStream(temporalName);` which makes me thing whether the argument above also applies to `fs.createWriteStream` 🤔 . What do you think?